tools/releaseBuild/azureDevOps/templates/nuget.yml

```diff
@@ -0,0 +1,154 @@
+parameters:
+  parentJobs: []
+
+jobs:
+- job: build_nuget
+  dependsOn:
+    ${{ parameters.parentJobs }}
+  displayName: Build NuGet packages
+  condition: succeeded()
+  pool: PowerShell
```

Comment:
Can you try this on the PkgES pool?  We should avoid using `PowerShell` unless we have to.  if we have to and it's not because of docker.  Can you add a comment as to why?

Reply:
Will update to PkgES